recipe/meta.yaml

```diff
@@ -204,17 +156,9 @@ outputs:
       imports:
         - xgboost
 
-  - name: _r-xgboost-mutex
-    version: 2.0
-    build:
-      string: {{ "cpu" if cuda_compiler == "None" else "gpu" }}_0
-    test:
-      commands:
-        - exit 0
-
   - name: r-xgboost
     script: >-
-      {{ "install-r-xgboost.sh" if not win else "install-r-xgboost.bat" }}
+      {{ "install-r-xgboost.sh" if not win else "install-win-wrapper.bat" }}
```

Comment:
From offline discussion with Jiaming, he recommended we use the same install process with Windows R that we use for Unix R. So switched Windows to use the wrapper script that we use for Windows builds of C++ & Python. That way, it will wrap the same Unix R install step with the tweaks needed for Windows to work

Reply:
Is it possible to retain `install-r-xgboost.bat`? Seems like this might be the cause for the Make error `make: *** No rule to make target '$(CXX17STD)''.  Stop.`

Reply:
Jiaming had suggested we use the same build process as upstream. This amounts to the wrapper Batch script calling bash script `install-r-xgboost.sh`. So we should no longer need `install-r-xgboost.bat` when things are working. Hopefully that makes the build here more canonical and representative of upstream, which seems desirable in the long run